bitnami/solr/templates/scripts-configmap.yaml

```
@@ -27,8 +32,7 @@ data:
         rm "/tmp/keystore.p12"
         keytool -import -file "/certs/ca.crt" -keystore "/opt/bitnami/solr/certs/truststore.p12" -storepass "${SOLR_SSL_TRUST_STORE_PASSWORD}" -noprompt
     else
-        cp "/certs/keystore.p12" "/opt/bitnami/solr/certs/keystore.p12"
-        cp "/certs/truststore.p12" "/opt/bitnami/solr/certs/truststore.p12"
+        echo "no certificate files provided ... nothing to do ..."
```

Could you change this by the following ? This would make all the log look similar.

```suggestion
        info "No certificate files provided ... nothing to do ..."
```
To use `info`, you will need to add the following script at the beginning of `init-certs.sh`
```
    . /opt/bitnami/scripts/liblog.sh
```

---

Ah! I did not notice that logging lib :+1: ... Thanks for the suggestion. I will push a change commit.